src/psyclone/tests/psyir/nodes/array_reference_test.py

```diff
@@ -139,9 +139,9 @@ def test_array_create_invalid2():
                 Literal("1", INTEGER_SINGLE_TYPE)]
     with pytest.raises(GenerationError) as excinfo:
         _ = ArrayReference.create(symbol_temp, children)
-    assert ("the symbol 'temp' should have the same number of dimensions as "
-            "indices (provided in the 'indices' argument). Expecting '3' but "
-            "found '1'." in str(excinfo.value))
+    assert ("the indices argument has '3' elements, but it must have a less "
+            "or equal number of dimensions than the 'temp' shape, which has "
```

Comment:
"than the shape of 'temp' which has..."

Reply:
I made the change you suggested in the array_reference.py instead, is that enough?

Reply:
Not quite (and sorry for spreading my comments around): "...'temp' shape" sounds like we have a shape named 'temp' but we want to say "shape of 'temp'".

Reply:
Done